api/src/main/java/org/apache/cloudstack/api/command/user/loadbalancer/UpdateLoadBalancerRuleCmd.java

```diff
@@ -66,6 +67,9 @@ public class UpdateLoadBalancerRuleCmd extends BaseAsyncCustomIdCmd {
     @Parameter(name = ApiConstants.PROTOCOL, type = CommandType.STRING, description = "The protocol for the LB")
     private String lbProtocol;
 
+    @Parameter(name = ApiConstants.CIDR_LIST, type = CommandType.LIST, collectionType = CommandType.STRING, description = "the cidr list to forward traffic from")
```

Comment:
`since` needs to be added

if `since` is `4.19.4`, but it is actually unsupported in 4.20.0 and 4.20.1 (which have been released), it might cause misunderstanding

@CodeBleu 

btw: I have tested the changes. the main problem is, which version should be targeted.
For PRs with api changes or db changes, I suggest to target to latest branch (`main`)